### PR TITLE
vgwort: Some fixes.

### PR DESCRIPTION
**Pull request by th-h**
* Change type of UNIQUE columns to VARCHAR. 
* Requires at least s9y 2.2.1-alpha1, s9y/Serendipity#511
* Fix a typo ("==" must be "=" in MySQL).
* Fix a typo (string concatenation is ".=", not "+=").

### CODE REVIEW

File: serendipity_event_vgwort/serendipity_event_vgwort.php [hunk 1]
```diff
@@ -18,7 +18,7 @@ function introspect(&$propbag) {
         $propbag->add('author',        'Malte Paskuda');
         $propbag->add('version',       '0.3.2');
         $propbag->add('requirements',  array(
-            'serendipity' => '2.1'
+            'serendipity' => '2.2.1'
```
Comment (onli) [inline serendipity_event_vgwort/serendipity_event_vgwort.php]: :+1:

File: serendipity_event_vgwort/serendipity_event_vgwort.php [hunk 2]
```diff
@@ -143,7 +143,7 @@ function unused() {
     function getCounter($entry_id) {
         global $serendipity;
         $sql = "SELECT counter_public, counter_private FROM {$serendipity['dbPrefix']}vgwort
-                WHERE entry_id == " . (int) $entry_id;
+                WHERE entry_id = " . (int) $entry_id;
```
Comment (onli) [inline serendipity_event_vgwort/serendipity_event_vgwort.php]: Oh, right.

File: serendipity_event_vgwort/serendipity_event_vgwort.php [hunk 3]
```diff
@@ -46,8 +46,8 @@ function setupDB() {
         global $serendipity;
         $sql = "CREATE TABLE  IF NOT EXISTS {$serendipity['dbPrefix']}vgwort (
                     entry_id INTEGER,
-                    counter_public TEXT UNIQUE NOT NULL,
-                    counter_private TEXT UNIQUE NOT NULL);";
+                    counter_public VARCHAR(32) UNIQUE NOT NULL,
```
Comment (onli) [inline serendipity_event_vgwort/serendipity_event_vgwort.php]: This  change I do not like. What if the counter gets bigger? Why not just keep it a TEXT data type?

Comment (th-h) [reply serendipity_event_vgwort/serendipity_event_vgwort.php]: It's not possible to use TEXT and UNIQUE in MySQL, see e.g. https://techjourney.net/mysql-error-1170-42000-blobtext-column-used-in-key-specification-without-a-key-length/

I think it's more important to keep UNIQUE than the TEXT type?

**Comment by onli** [comment]
I just saw your blog post, I ahd forgotten about this. Sorry! Merging now.

**Pull request merged**